settings.gradle

```diff
@@ -0,0 +1,10 @@
+plugins {
+    id 'com.diffplug.blowdryerSetup' version '1.6.0'
+}
+
+apply plugin: 'com.diffplug.blowdryerSetup'
+
+blowdryerSetup {
+    //github('GTNewHorizons/ExampleMod1.7.10', 'tag', '0.1.4')
+    devLocal '.' // Use this when testing config updates locally
```

Comment:
Please remove spotless.gradle file and change that to github

Reply:
Why remove spotless.gradle? It won't work without it.

Reply:
See the example file or any other repo migrated.  You copied it wrong.  It should be pointing to GitHub.